src/benchmarks/micro/runtime/System.Reflection/Invoke.cs

```diff
@@ -83,80 +88,137 @@ public void Method0_NoParms()
         // Include the array allocation and population for a typical scenario.
         public void StaticMethod4_arrayNotCached_int_string_struct_class()
         {
-            object[] args = new object[] { 42, "Hello", default(MyBlittableStruct), s_MyClass };
-            s_method_int_string_struct_class.Invoke(null, args);
+            for (int i = 0; i < Iterations; i++)
+            {
+                object[] args = new object[] { 42, "Hello", default(MyBlittableStruct), s_MyClass };
+                s_method_int_string_struct_class.Invoke(null, args);
+            }
         }
 
         [Benchmark]
```

Comment:
```suggestion
        [Benchmark(OperationsPerInvoke = Iterations)]
```
? (to get the time per iteration and not the total time taken for iteration**s**)

(same on other places as well)

Reply:
Isn't it better than manual loop?

Reply:
The manual loop should still be there. BDN scales the result by the number in `OperationsPerInvoke`, so the results are "per iteration".

Reply:
I added this. Here's the results I get vs 6.0:


|                                              Method |       Job | Runtime |Toolchain |     Mean |    Error |   StdDev |
|--------------------------------------------------- |----------- |--------- |---------- |----------:|----------:|----------:|
|StaticMethod4_arrayNotCached_int_string_struct_class |Job-WSRKIS |.NET 6.0 |   net6.0 |299.86 ns | 6.566 ns | 7.026 ns |
|StaticMethod4_arrayNotCached_int_string_struct_class |Job-YYAVGW |.NET 7.0 |   net7.0 | 87.78 ns | 2.851 ns | 3.283 ns |
|                                                     |           |         |          |          |          |          |
|               StaticMethod4_int_string_struct_class |Job-WSRKIS |.NET 6.0 |   net6.0 |299.36 ns |25.425 ns |29.279 ns |
|               StaticMethod4_int_string_struct_class |Job-YYAVGW |.NET 7.0 |   net7.0 | 65.93 ns | 0.877 ns | 0.732 ns |
|                                                     |           |         |          |          |          |          |
|   StaticMethod4_ByRefParams_int_string_struct_class |Job-WSRKIS |.NET 6.0 |   net6.0 |580.54 ns |24.741 ns |26.473 ns |
|   StaticMethod4_ByRefParams_int_string_struct_class |Job-YYAVGW |.NET 7.0 |   net7.0 |278.42 ns |11.915 ns |13.244 ns |
|                                                     |           |         |          |          |          |          |
|                                      Ctor0_NoParams |Job-WSRKIS |.NET 6.0 |   net6.0 | 98.22 ns | 3.625 ns | 3.878 ns |
|                                      Ctor0_NoParams |Job-YYAVGW |.NET 7.0 |   net7.0 | 23.04 ns | 2.437 ns | 2.807 ns |
|                                                     |           |         |          |          |          |          |
|              Ctor0_ActivatorCreateInstance_NoParams |Job-WSRKIS |.NET 6.0 |   net6.0 | 14.54 ns | 0.443 ns | 0.493 ns |
|              Ctor0_ActivatorCreateInstance_NoParams |Job-YYAVGW |.NET 7.0 |   net7.0 | 13.03 ns | 0.460 ns | 0.511 ns |
|                                                     |           |         |          |          |          |          |
|                       Ctor4_int_string_struct_class |Job-WSRKIS |.NET 6.0 |   net6.0 |298.27 ns |16.889 ns |18.773 ns |
|                       Ctor4_int_string_struct_class |Job-YYAVGW |.NET 7.0 |   net7.0 | 73.24 ns | 2.636 ns | 2.929 ns |
|                                                     |           |         |          |          |          |          |
|                       Ctor4_ActivatorCreateInstance |Job-WSRKIS |.NET 6.0 |   net6.0 |799.71 ns |26.278 ns |30.262 ns |
|                       Ctor4_ActivatorCreateInstance |Job-YYAVGW |.NET 7.0 |   net7.0 |434.39 ns |14.795 ns |16.444 ns |
|                                                     |           |         |          |          |          |          |
|                                    Property_Get_int |Job-WSRKIS |.NET 6.0 |   net6.0 |126.61 ns | 8.056 ns | 8.955 ns |
|                                    Property_Get_int |Job-YYAVGW |.NET 7.0 |   net7.0 | 22.34 ns | 0.422 ns | 0.433 ns |
|                                                     |           |         |          |          |          |          |
|                                  Property_Get_class |Job-WSRKIS |.NET 6.0 |   net6.0 | 71.95 ns | 2.306 ns | 2.563 ns |
|                                  Property_Get_class |Job-YYAVGW |.NET 7.0 |   net7.0 | 19.22 ns | 0.424 ns | 0.472 ns |
|                                                     |           |         |          |          |          |          |
|                                    Property_Set_int |Job-WSRKIS |.NET 6.0 |   net6.0 |129.85 ns | 4.446 ns | 5.120 ns |
|                                    Property_Set_int |Job-YYAVGW |.NET 7.0 |   net7.0 | 45.50 ns | 0.909 ns | 0.892 ns |
|                                                     |           |         |          |          |          |          |
|                                  Property_Set_class |Job-WSRKIS |.NET 6.0 |   net6.0 |112.52 ns | 5.777 ns | 6.422 ns |
|                                  Property_Set_class |Job-YYAVGW |.NET 7.0 |   net7.0 | 42.47 ns | 1.195 ns | 1.376 ns |
|                                                     |           |         |          |          |          |          |
|                                       Field_Get_int |Job-WSRKIS |.NET 6.0 |   net6.0 | 67.79 ns | 2.411 ns | 2.776 ns |
|                                       Field_Get_int |Job-YYAVGW |.NET 7.0 |   net7.0 | 68.09 ns | 2.805 ns | 3.230 ns |
|                                                     |           |         |          |          |          |          |
|                                     Field_Get_class |Job-WSRKIS |.NET 6.0 |   net6.0 | 39.11 ns | 1.266 ns | 1.243 ns |
|                                     Field_Get_class |Job-YYAVGW |.NET 7.0 |   net7.0 | 46.92 ns | 1.200 ns | 1.232 ns |
|                                                     |           |         |          |          |          |          |
|                                       Field_Set_int |Job-WSRKIS |.NET 6.0 |   net6.0 | 72.87 ns | 2.699 ns | 3.108 ns |
|                                       Field_Set_int |Job-YYAVGW |.NET 7.0 |   net7.0 | 71.95 ns | 2.202 ns | 2.356 ns |
|                                                     |           |         |          |          |          |          |
|                                     Field_Set_class |Job-WSRKIS |.NET 6.0 |   net6.0 | 69.97 ns | 2.188 ns | 2.432 ns |
|                                     Field_Set_class |Job-YYAVGW |.NET 7.0 |   net7.0 | 71.58 ns | 2.173 ns | 2.503 ns |

Reply:
Thanks. Added "OperationsPerInvoke"